Sources/MapboxNavigation/NavigationViewHelper.swift

```diff
@@ -0,0 +1,170 @@
+
+import MapboxDirections
+import MapboxCoreNavigation
+import CoreLocation
+import UIKit
+
+
+class NavigationViewHelper: NavigationComponent, ViewCycleObserver {
```

Comment:
With this name, NavigationViewHelper is likely to become a bit of a junk drawer in the future. Let’s see if we can find more appropriate homes for each of its members.

If we whittle this class down to just the methods involved with runtime styling, then it would be more reusable like CameraController and we can potentially make it public. That was the original intent behind RouteMapViewController, before it gained a lot of other features that weren’t as closely tied to the map view.

Reply:
In result, it turns out that it only handles events related to Route Line drawing and it's related features, so I've documented and renamed it